package.json

```diff
@@ -43,13 +43,11 @@
     "babel-loader": "^8.2.2",
     "babel-plugin-rewire": "1.2.0",
     "babel-plugin-syntax-trailing-function-commas": "^6.8.0",
-    "bundle-loader": "0.5.6",
     "css-loader": "^5.2.7",
     "eslint": "7",
     "eslint-config-prettier": "6.10.0",
     "except": "^0.1.3",
     "faker": "^5.5.3",
-    "file-loader": "4.2.0",
```

Comment:
`file-loader` is deprecated. Webpack recommends to use asset modules instead.
See https://v4.webpack.js.org/loaders/file-loader/